.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -675,6 +675,11 @@ stages:
         baseImage: $(baseImage)
         command: "BuildLinuxIntegrationTests --framework $(publishTargetFramework)"
 
+    - task: DownloadPipelineArtifact@2
+      displayName: Download linux tracer home
+      inputs:
+        artifact: linux-tracer-home-$(baseImage)
+        path: $(System.DefaultWorkingDirectory)/tracer/serverlessArtifacts
```

Comment:
This _shouldn't_ be necessary. The home directory should already be restored as part of `steps/restore-working-directory.yml` (It's in `$(tracerHome)`)